yktrace/src/hwt/mapper.rs

```diff
@@ -18,15 +18,18 @@ use ykutil::addr::code_vaddr_to_off;
 const BLOCK_MAP_SEC: &str = ".llvm_bb_addr_map";
 static BLOCK_MAP: Lazy<BlockMap> = Lazy::new(|| BlockMap::new());
 
+/// Indicates that (in LLVM) there was no BasicBlock corresponding with a MachineBasicBlock.
+const NO_BASICBLOCK: u64 = u64::MAX;
+
 /// The information for one basic block, as per:
 /// https://llvm.org/docs/Extensions.html#sht-llvm-bb-addr-map-section-basic-block-address-map
 #[derive(Debug)]
 #[allow(dead_code)]
 struct BlockMapEntry {
     /// Function offset.
     f_off: u64,
-    /// Basic block number.
-    bb: usize,
+    /// Basic block number or None if there is no corresponding block.
+    bb: Option<usize>,
```

Comment:
The `Option` will mean this attribute takes 16 bytes on a 64-bit machine: shouldn't we make this a `u64` and use the same `NO_BASICBLOCK` trick?

Reply:
We can do yeah.